### PR TITLE
Retune archery draw weights

### LINKED ISSUE #38642: Bow/crossbow strength requirements are unrealistic
# Describe the bug

The recent archery update seriously brought up both damage and strength requirements for bows and crossbows. I dunno about damage, buth the STR requirements seem very unrealistic. I keep in mind design values - an average human has 8 strength, a strong human has 10, a very strong person has 12, a top-tier athlete has 14. Look:

1. A 205 crossbow/200 lb composite crossbow requires 12 strength to draw, compound crossbow requires 11. [Here](https://www.youtube.com/watch?v=2IdfmaC_t-Q) we have a man, drawing about 150 lbs with his bare hands, 300 lbs in sitting span and even more with very basic devices (**1500** lbs with a windlass, so the ingame heavy crossbow can be easily upped from the current 800 lbs). He's really not looking like a world-class athlete for me.
2. [Here's](https://www.youtube.com/watch?v=2IdfmaC_t-Q) another average man drawing crossbows. Yes, he's using basic devices - goats foot lever, the spanning belt and the gaff lever, but they're like REALLY basic ones - every survivor should be able to use one. He claims you're able to draw about **450 lbs** this way.
3. [Here's](https://www.youtube.com/watch?v=ohutwHqClOc) the guy drawing 80 lbs bows. 80 lb in-game bow requires 12 (!) strength, and the guy from the video is seriously not looking like the athlete or heavylifter.
4. [Here's](https://www.youtube.com/watch?v=CoSCl_qcJsQ) a man with 110 lb bow (14 STR req). Yeah, he's definitely strong, but he's not a world-class powerlifter.
5. [Here's](https://www.youtube.com/watch?v=-KokNtdh4_Y) a regular man easily drawing a 60 lb bow (10 STR req). Is he really about a half-way to the olympic championship?
6. [Here's](https://www.youtube.com/watch?v=MjY2QrU4sm4) a strong man indeed drawing 210 lb bow. Ingame wooden 145 lb greatbow requires 19 (!) STR. Man from the video is not a cyborg or mutant, I'm sure.
7. [Here's](https://www.youtube.com/watch?v=xrzPFIjAZdU) a 100 lb bow. 
8. [Here's](https://www.youtube.com/watch?v=WmhYwboqBOg) a 90 lb one. 

There are lots of evidences, I think you got the point - both bows and crossbows, despite the great drawing weight, do not require tremendous physical strength as they require now.

# Steps To Reproduce

1. Start as an average person with 8-9 strength.
2. Spawn a compound bow (high), composite bow, reflex bow, hybrid longbow, crossbow or composite crossbow.
3. There's no way to shoot with them.

# Expected behavior

Able to shoot from a bow or crossbow. A bow/crossbow with req 12 or even 14 STR is probably a ballista or similar siege machine. I don't touch damage or range, because they are calculated by some complex formulas, but STR req is set arbitrary, so it can be changed more easily.

The lower cap is currently 30 lbs with 10 lb/1 STR. I suppose to remove the cap - straight 10lb/1 STR would be fine: i.e. 6 STR person could draw a string of a 60-lb hunting bow, but stronger person would still benefit in bonus range. 8 STR person could draw a 80-lb bow, etc. 

Self bow - 30 lbs - I guess even a feeble person or a kid can draw 30 lbs.
Short bow - 50 lbs - maybe leave it as is at 6 STR, I guess, or lower to 5.
Compound bow (low) - 40 lbs - it seems even a feeble person is able to draw it.
Compound bow (normal) - 60 lbs - lower the requirement to 6 STR.
Compound bow (high) - 80 lbs - to 8 STR. Compound bows are very easy to draw thanks to a pulley system.
Composite bow - 110 lbs - to 11 STR.
Recurve bow/Reflex bow - 60 lbs - to 6 STR.
Longbow - 80 lbs - to 8 STR.
Wooden greatbow - 145 lbs - to 14 STR. Maybe 15 if it strictly designed for superhumans only.
Compound greatbow - 120 lbs - to 12 STR.
Reflex recurve bow - 50 lbs - to 5 STR.

8 STR max for all crossbows (and even without the req at all with drawing devices).

TL;DR: 8-9 STR should be more than enough for most string weapons, more for the heaviest ones only.

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
SUMMARY: None

#### Purpose of change
Fixes #38642 
Many NPCs and players became unable to use most crossbows due to their strength requirements.
Some bows had too-high draw weights and were unusable for many characters.
The composite bow, an ancient and obscure warbow, was spawning all over the place in modern New England.
The mapping from draw weight to strength was too agressive.

#### Describe the solution
Rebalance crossbows around lowered strength requirements.
Also some incidental fixes like removing steel material from bows.
Drop bow draw weight requirements based on feedback and reexamining a number of sources.
Remove spawns of composite warbow and other non-existent bows.

#### Describe alternatives you've considered
Longer term, the crossbow draw weights can go back up and be achievable based on reloading aids.
After 0.E, the available bows need to be entirely overhauled because the current selection of bows is really bizarre.
The archery code should be able to handle bows with variant versions that have different strength requirements.

#### Testing
This might break some of the damage tests.
Confirm that crossbow armed NPCs can use their crossbows.

#### Additional context
See #37241

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
Were the reloading mechanisms from #35971 incomplete and can't be added now?

**Comment by ZhilkinSerg** [comment]
Archery tests probably need to be updated now.

**Comment by FuelType-Memes** [comment]
You drop the req for `crossbow` by 4 and drop damage by 10, then you drop the req for `compositecrossbow` by 2 and drop the damage by 12, so now:

Crossbow with
`100lb @ 12in powerstroke, 34in wood limbs, 67.8J` has 17 DMG, while

Composite crossbow
`160lb draw, 12in draw, 120J` has 20 DMG

What are those arbitrary numbers? Why does nearly twice the energy equate to only a +3 to DMG?

**Comment by anothersimulacrum** [comment]
From `doc/GAME_BALANCE.md`:
> ## Archery damage
> Bow damage is based on the momentum achieved in the projectile.  Since arrows and bolts have sharp cutting surfaces, the penetration and therefore damage achieved is based on the projectile's capacity for slicing through tissues.  The arrow has a modifier based on construction, material and design, most critically centered around the effectiveness of the head.  Base damage is calculated from momentum by taking momentum in Slug-foot-seconds, multiplying by 150 and subtracting 32. This was arrived at by taking well-regarded bowhunting guidelines and determining the damage numbers necessary for a kill of various game on a critical hit, see tests/archery_damage_test.cpp for details.

I haven't done the calculations to check that it matches, but you can if you'd like.

**Comment by FuelType-Memes** [comment]
> I haven't done the calculations to check that it matches, but you can if you'd like.

Thank you, my mistake for not checking that! Yes, the calculations are correct, and even though I still disagree about the damage it's consistent with the doc:
- crossbow: 0.327 * 150 - 32 = 17.05
- composite crossbow: 0.35 * 150 - 32 = 20.5

@kevingranade there is no info about rounding in the doc, what was your intent? `floor` or `round`?

**Comment by Amoebka** [comment]
I would suggest dropping survivor-made Self Bow down to 4 strength or no requirement at all (and reducing its power if necessary). As per its description, it's "made specifically for the person using it", therefore it should be usable on all characters (as they can just adjust the draw weight until it's manageable for them). The idea behind it has also been "a garbage bow literally everyone can use", while currently characters below 6 strength can't use any bows at all.

**Comment by kevingranade** [comment]
Any lower damage and the selfbow will be literally useless.  I didn't change the description or name because I don't want to make more work for translators, the description will be changing after 0.E.

**Comment by Hulkninja** [comment]
I still need a strength of 12 to use a compound bow..?

This isn't right. I've got a friend who practices archery, he's a pretty skinny dude and he can use any type of bow there is just fine. Strength requirements for bows still need to be dropped. Or better yet, just removed entirely, since most people CAN use any bow IRL just fine if they have some experience. If my character has level 5 archery, a strength requirement of 12 is just absurd. It'd be more reasonable to put the strength requirement at 8, and an archery requirement at 4, or maybe even a bit higher.

Tl;Dr:
Anyone who's not physically disabled should be able to use all but the most outlandish bows with a bit of practice, because that's how it works IRL. I don't want to have to spec into strength like crazy when I'm trying to do an archery build.
